packages/website/src/bin/default-config.js

```diff
@@ -1,10 +1,15 @@
 const identityFunc = require('lodash.identity');
 
 module.exports = {
-  docs: './docs',
+  docs: {
+    path: './docs',
+    name: 'Docs',
+    description: 'This is a sample documentation for the website.',
+  },
   packages: ['./packages/*'],
   useManifests: false,
   siteName: 'Brisk Docs',
   webpack: identityFunc,
   showSubExamples: false,
+  packagesDescription: 'This is a sample package for the documentation website',
```

Comment:
Similar, here we should have something like "View documentation about individual packages"